src/pages/workspace/distanceRates/PolicyDistanceRatesSettingsPage.tsx

```diff
@@ -78,37 +89,72 @@ function PolicyDistanceRatesSettingsPage({policy, policyCategories, route}: Poli
                 testID={PolicyDistanceRatesSettingsPage.displayName}
             >
                 <HeaderWithBackButton title={translate('workspace.common.settings')} />
-                <View style={styles.flexGrow1}>
-                    <OfflineWithFeedback
-                        errors={ErrorUtils.getLatestErrorField(customUnits[customUnitID] ?? {}, 'attributes')}
-                        pendingAction={customUnits[customUnitID]?.pendingFields?.attributes}
-                        errorRowStyles={styles.mh5}
-                        onClose={() => clearErrorFields('attributes')}
-                    >
-                        <UnitSelector
-                            label={translate('workspace.distanceRates.unit')}
-                            defaultValue={defaultUnit}
-                            wrapperStyle={[styles.ph5, styles.mt3]}
-                            setNewUnit={setNewUnit}
-                        />
-                    </OfflineWithFeedback>
-                    {policy?.areCategoriesEnabled && OptionsListUtils.hasEnabledOptions(policyCategories ?? {}) && (
+                <ScrollView contentContainerStyle={styles.flexGrow1}>
+                    <View style={styles.flexGrow1}>
```

Comment:
This might be redundant as we already have contentContainerStyle={styles.flexGrow1} for the `ScrollView`.
Why do we need `styles.flexGrow1` here?